### PR TITLE
security: Upgrade to Python 3.6.13

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Upgrading for security patches

### CODE REVIEW

**Review by wedamija — APPROVED** [review]

**Comment by BYK** [comment]
Should we pin the patch version in our Docker files too: https://github.com/getsentry/sentry/blob/9e32d07198abc69f65c0268418f267e269d3b256/docker/Dockerfile#L1 ?

**Comment by joshuarli** [comment]
> Should we pin the patch version in our Docker files too:
> 
> https://github.com/getsentry/sentry/blob/9e32d07198abc69f65c0268418f267e269d3b256/docker/Dockerfile#L1
> ?

Yeah, that'd be preferable.

**Comment by joshuarli** [comment]
So uhh, I think this failing test case is just not right.

```
_______________________________________________________ BuildIncidentAttachmentTest.test_parse_link ________________________________________________________
tests/sentry/integrations/slack/test_utils.py:331: in test_parse_link
    assert (
E   AssertionError: assert 'organization...esolved%27%5D' == 'organizations...esolved%27%5D'
E     Skipping 59 identical leading characters in diff, use -v to show
E     - roject%7D&amp%3Bquery=%5B%27is%3Aunresolved%27%5D
E     ?           ------
E     + roject%7D&query=%5B%27is%3Aunresolved%27%5D
```

`&amp;` is an html or xml encoding. URL encoding would be `%26` if you wanted a literal ampersand and not a query parameter.

**Comment by EvanPurkhiser** [comment]
You're right. Just change it

**Comment by EvanPurkhiser** [comment]
Approved

File: tests/sentry/integrations/slack/test_utils.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def test_build_group_attachment_color_warning(self):
         assert build_group_attachment(warning_event.group, warning_event)["color"] == "#FFC227"
 
     def test_parse_link(self):
-        link = "https://meowlificent.ngrok.io/organizations/sentry/issues/167/?project=2&amp;query=is%3Aunresolved"
+        link = "https://meowlificent.ngrok.io/organizations/sentry/issues/167/?project=2&query=is%3Aunresolved"
```
Comment (chadwhitacre) [inline tests/sentry/integrations/slack/test_utils.py:328]: 👀

Comment (chadwhitacre) [reply tests/sentry/integrations/slack/test_utils.py:328]: Found https://github.com/getsentry/sentry/pull/23969#issuecomment-786328094 ftr.

**Review by chadwhitacre — APPROVED** [review]
Only thing an audit (`ag 3\\.6`) turned up is `mypy.ini`. That doesn't want a patch version, right?

**Review by armenzg — APPROVED** [review]

**Review by BYK — APPROVED** [review]

**Pull request merged**